client/src/pages/Login.js

```diff
@@ -0,0 +1,235 @@
+    height: "100vh",
+  },
+  img: {
+    margin: "auto",
+    display: "block",
+    maxWidth: "100%",
+    maxHeight: "100%",
+  },
+  text: {
+    paddingTop: "0.5rem",
+    paddingLeft: "5%",
+    paddingRight: "5%",
+  },
+  account: {
+    paddingTop: "0.5rem",
+  },
+  button: {
+    borderRadius: 35,
+    marginLeft: "1rem",
+    marginRight: "0rem",
+    marginBottom: "2rem",
+    paddingLeft: "1.5rem",
+    paddingRight: "1.5rem",
+    minWidth: "6.2rem",
+  },
+  start: {
+    paddingBottom: "1.5rem",
+    fontFamily: "Arial",
+  },
+  input: {
+    marginTop: "0.5rem",
+    marginBottom: "1rem",
+  },
+  continue: {
+    borderRadius: 35,
+    marginTop: "1rem",
+    paddingLeft: "1.5rem",
+    paddingRight: "1.5rem",
+  },
+});
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+      openSnack: false,
+      severity: "error",
+      message: "Invalid email or password!",
+    };
+  }
+
+  componentDidMount() {
+    fetch("/welcome")
+      .then((res) => {
+        console.log(res);
+        if (res.status === 200) return res.json();
+        else throw Error("Couldn't connect to the server");
+      })
+      .then((res) => {})
+      .catch((err) => {
+        console.log(err.message);
+      });
+  }
+
+  onChangeEmail = (e) => {
+    this.setState({ email: e.target.value });
+  };
+  onChangePassword = (e) => {
+    this.setState({ password: e.target.value });
+  };
+  signUp = () => {
+    this.props.history.push({
+      pathname: "/signup",
+      state: {},
+    });
+  };
+  handleClick = () => {
+    this.validation();
+    this.setState({ openSnack: true });
+  };
+  async validation() {
+    //fetch here
+    const res = fetch("/signin", {
+      method: "post",
+      headers: { "Content-Type": "application/json" },
+      body: {
+        email: this.state.email,
+        password: this.state.password,
+      },
+    })
+      .then((response) => response.json())
+      .then((responseJson) => {
+        return responseJson.cookie;
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+
+    var success = false;
+    if (true) {
+      success = true;
+    }
+    await this.setState({
+      severity: success ? "success" : "error",
+      message: success ? "Sign in successfully!" : "Invalid email or password!",
+    });
+    if (success === true) {
+    }
+  }
+  handleClose = (event, reason) => {
+    if (reason === "clickaway") {
+      return;
+    }
+    this.setState({ openSnack: false });
+  };
+
+  render() {
+    const { classes } = this.props;
+    return (
+      <div className={classes.root}>
+        <Paper className={classes.paper} elevation={0}>
+        <Grid class="frame" container spacing={0}>
+            <Grid item>
+              <Grid class="image">
+                <img className={classes.img} alt="complex" src={(width < 680) ? pic2 : (width > 1100) ? pic1 : pic3}/>
```

Comment:
it's not great practice to have three different images, if they are the same image with just different aspect ratios. Instead you should be using a background image or be changing the CSS depending on the screen size.

Reply:
> it's not great practice to have three different images, if they are the same image with just different aspect ratios. Instead you should be using a background image or be changing the CSS depending on the screen size.

Can I use @media in the React Hooks Style API with material-ui?

Reply:
Yes, look at my post above (specifically at https://material-ui.com/customization/breakpoints/)
For a whole guide, see https://material-ui.com/guides/responsive-ui/

Reply:
Use breakpoints to assign different widths to the image